### PR TITLE
Fix deployment conflict with live content

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes: #1305

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/ocp.py [hunk 1]
```diff
@@ -182,6 +185,9 @@ def get(
             dict: Dictionary represents a returned yaml file
         """
         resource_name = resource_name if resource_name else self.resource_name
+        selector = selector if selector else self.selector
+        if selector:
+            resource_name = ""
```
Comment (RazTamir) [inline ocs_ci/ocs/ocp.py]: I wonder if it breaks backward compatibility.
If we are 'get'ing with name AND selector, it should work too right?

Comment (petr-balogh) [reply ocs_ci/ocs/ocp.py]: Actually this is behavior of OC client:
```console
$ oc get packagemanifest -n openshift-marketplace --selector "ocs-operator-dev=true" ocs-operator
error: name cannot be provided when a selector is specified
```
You are not able to use selector and the name of the resource at the same time.   

I didn't come out with better solution here.  We basically prefer selector over the name here.

I see the deployment passed here:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3827/console

I am running also whole tier1 to see if I didn't break anything.

Comment (RazTamir) [reply ocs_ci/ocs/ocp.py]: Ah ok.. Didn't know that (:
LGTM

**Comment by petr-balogh** [comment]
Verification job running here: https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3827/console . 

Cause of temp issue in travis-ci I had to force push last commit again to trigger travise once again.

**Comment by petr-balogh** [comment]
The job failed cause of jenkins got stuck and restarted. But when I connected to jenkins slave I saw that pytest finished. And results of xunit are available here:
```xml
<testsuite errors="0" failures="0" hostname="temp-jslave-jenkins-pr1325" name="jenkins-pr1325" skipped="1" tests="49" time="19782.847" timestamp="2020-01-16T12:56:00.311246">
```
So I see the results looks good. Marking as verified

**Comment by petr-balogh** [comment]
Btw results are visible here:
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/jenkins-pr1325/jenkins-pr1325_20200116T114038/logs/test_results.xml

**Pull request merged**